include/substrait-mlir/Dialect/Substrait/IR/SubstraitOps.td

```diff
@@ -525,24 +526,35 @@ def Substrait_ProjectOp : Substrait_RelOp<"project", [
   }];
 }
 
-def Substrait_UnionDistinctOp : Substrait_RelOp<"union_distinct", [
+def SetOpKind : I32EnumAttr<"SetOpKind", "The enum values correspond to those in the SetRel.SetOp message.",[I32EnumAttrCase<"unspecified", 0>,
```

Comment:
Nit: break into more lines (of ~80 characters, I think?).

Reply:
The issue is that the description, "The enum values...", is a single parameter within the constructor for the class I32EnumAttr<string name, string summary, list<I32EnumAttrCase> cases>. I think it cannot be split into multiple strings without breaking the integrity of the function call. @ingomueller-net

Reply:
You have several options: (1) break the line before the string and (2) use a multline string (`[{multiline string}]`).

Reply:
My bad! I tried with """ """ and it did not work and I forgot about the [{ syntax! Thanks!